src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -226,60 +226,116 @@ bool CallstackThreadBar::IsEmpty() const {
   return callstack_count == 0;
 }
 
-[[nodiscard]] std::string CallstackThreadBar::SafeGetFormattedFunctionName(
-    const CallstackInfo& callstack, size_t frame_index, int max_line_length) const {
+CallstackThreadBar::ModuleAndFunctionNameToFormat CallstackThreadBar::SafeGetModuleAndFunctionNames(
+    const CallstackInfo& callstack, size_t frame_index) const {
   ORBIT_CHECK(capture_data_ != nullptr);
   if (frame_index >= callstack.frames().size()) {
-    return std::string("<i>") + orbit_client_data::kUnknownFunctionOrModuleName + "</i>";
+    ModuleAndFunctionNameToFormat module_and_function_name_to_format;
```

Comment:
I added a static constant here. I don't really want to set these values with a default constructor as they are not just empty values. The static initialization uses a lambda because Visual Studio isn't happy about designated initializers. Note that we return by value and note that this case is unexpected so it makes not difference, but I guess conceptually it's a bit better. Let me know if you prefer something else.